okapi-core/src/test/java/org/folio/okapi/ProxyTest.java

```diff
@@ -2177,20 +2214,20 @@ public void testRequestOnly(TestContext context) {
       c.getLastReport().isEmpty());
 
     final String docRequestOnly = "{" + LS
-      + "  \"id\" : \"request-only-1.0.0\"," + LS
-      + "  \"name\" : \"request-only\"," + LS
-      + "  \"filters\" : [ {" + LS
-      + "    \"methods\" : [ \"GET\", \"POST\" ]," + LS
-      + "    \"path\" : \"/testb\"," + LS
-      + "    \"level\" : \"33\"," + LS
-      + "    \"type\" : \"request-only\"," + LS
-      + "    \"permissionsRequired\" : [ ]" + LS
-      + "  } ]," + LS
-      + "  \"launchDescriptor\" : {" + LS
-      + "    \"exec\" : "
-      + "\"java -Dport=%p -jar ../okapi-test-module/target/okapi-test-module-fat.jar\"" + LS
-      + "  }" + LS
-      + "}";
+        + "  \"id\" : \"request-only-1.0.0\"," + LS
+        + "  \"name\" : \"request-only\"," + LS
+        + "  \"filters\" : [ {" + LS
+        + "    \"methods\" : [ \"GET\", \"POST\" ]," + LS
+        + "    \"path\" : \"/testb\"," + LS
+        + "    \"level\" : \"30\"," + LS
```

Comment:
Just curious why?

Reply:
It is meant to be executed after auth (level=10) and and before pre(level=40).

Reply:
I get that, but by change from 33 to 30?